src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+    private ActorDao actorDao = new ActorDaoImpl(sessionFactory);
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
```

Comment:
This constructor is correct, but it's not used because you are creating a new instance of ActorDaoImpl in the field declaration. Please remove the instantiation of ActorDaoImpl from the field declaration and use this constructor to set the ActorDao.